pages/developers/pep621.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: page
+title: Pure Python packaging
+permalink: /developer/pep621
+nav_order: 4
+parent: Developer information
+---
+
+# Pure Python packaging
+
+Pure Python packages with simple builds, that is, no binary extensions, then
```

Comment:
This first sentene does not read well at all. It's super chatty.

Reply:
Redid the paragraph. Not sure what happened to this first sentence.